api/src/main/java/com/okta/sdk/client/ClientBuilder.java

```diff
@@ -431,6 +432,15 @@
      */
     ClientBuilder setRetryMaxAttempts(int maxAttempts);
 
+    /**
+     * Sets the {@link RequestExecutorFactory}, otherwise the one will be used from the classpath.
```

Comment:
"used from the classpath" should instead mention it's loaded as a Service / SPI via the `RequestExecutorFactory` class